src/main/resources/com/xpandit/plugins/xrayjenkins/model/ServerConfiguration/config.jelly

```diff
@@ -132,6 +132,7 @@
 			    
 			    Q(testConnectionButton).click(function(evt) {
 			        Q(evt.target).parents('.test-connection-button-wrapper').find('.error').remove();
```

Comment:
You can use several selectors in the same find:
`Q(evt.target).parents('.test-connection-button-wrapper').find('.error,.ok').remove();`